### PR TITLE
minor fixes for 64 bits

### PR DESCRIPTION
**Pull request by GermanAizek**

### CODE REVIEW

File: src/lib_json/json_writer.cpp [hunk 1]
```diff
@@ -350,7 +350,7 @@ static String valueToQuotedStringN(const char* value, unsigned length,
 }
 
 String valueToQuotedString(const char* value) {
-  return valueToQuotedStringN(value, static_cast<unsigned int>(strlen(value)));
+  return valueToQuotedStringN(value, static_cast<size_t>(strlen(value)));
```
Comment (BillyDonahue) [inline src/lib_json/json_writer.cpp:353]: strlen already returns size_t. Better to just remove the static_cast

File: src/lib_json/json_writer.cpp [hunk 2]
```diff
@@ -985,7 +985,7 @@ void BuiltStyledStreamWriter::writeValue(Value const& value) {
         Value const& childValue = value[name];
         writeCommentBeforeValue(childValue);
         writeWithIndent(valueToQuotedStringN(
-            name.data(), static_cast<unsigned>(name.length()), emitUTF8_));
+            name.data(), static_cast<size_t>(name.length()), emitUTF8_));
```
Comment (BillyDonahue) [inline src/lib_json/json_writer.cpp:988]: length() is already size_t, so cast can be removed entirely.

File: src/lib_json/json_writer.cpp [hunk 3]
```diff
@@ -422,8 +422,8 @@ void FastWriter::writeValue(const Value& value) {
       const String& name = *it;
       if (it != members.begin())
         document_ += ',';
-      document_ += valueToQuotedStringN(name.data(),
-                                        static_cast<unsigned>(name.length()));
+      document_ += 
+          valueToQuotedStringN(name.data(), static_cast<size_t>(name.length()));
```
Comment (BillyDonahue) [inline src/lib_json/json_writer.cpp:426]: remove cast

**Review by BillyDonahue — CHANGES_REQUESTED** [review]

File: src/lib_json/json_writer.cpp (hunk 1 above)
Comment (GermanAizek) [reply src/lib_json/json_writer.cpp]: fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b

File: src/lib_json/json_writer.cpp (hunk 2 above)
Comment (GermanAizek) [reply src/lib_json/json_writer.cpp]: fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b

File: src/lib_json/json_writer.cpp (hunk 3 above)
Comment (GermanAizek) [reply src/lib_json/json_writer.cpp]: @BillyDonahue, fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b

**Comment by BillyDonahue** [comment]
this looks good. I am mobile and can approve later, but looks good to me.

**Comment by BillyDonahue** [comment]
You're going to need to clang-format the changes to pass the TravisCI testing.
See the fail logs for details on what exactly needs to change.

**Comment by BillyDonahue** [comment]
LGTM.

PS: Please switch to normal git push without force. Force wipes out incremental diffs and context so a reviewer would have to review the whole change from scratch on every force push upload.

**Pull request merged**